geometry/eolearn/tests/test_morphology.py

```diff
@@ -58,8 +63,8 @@ def test_erosion_partial(self):
                 self.assertLessEqual(np.sum(mask_after == label), np.sum(mask_before == label),
                                      msg="error in the erosion process")
             else:
-                self.assertEqual(np.sum(mask_after == label), np.sum(mask_before == label),
-                                 msg="error in the erosion process")
+                self.assertTrue(np.array_equal(mask_after == label, mask_before == label),
+                                msg="error in the erosion process")
```

Comment:
Would it make sense to also add a test here, which tests that the partial erosion is not the same as the full erosion?

Reply:
Ignore this as well.